src/app/restaurant/restaurant.component.html

```diff
@@ -4,4 +4,46 @@
   <p>Description: {{ restaurant.description }}</p>
 
   <p>Date of Creation: {{ restaurant.dateOfCreation }}</p>
+  <form [formGroup]="reviewForm">
+    <label for="review">Add review:</label>
```

Comment:
Label has a for attribute but there is no element with `id="review"`

Reply:
actually there is no need for label because it doesn't work for stars?

Reply:
We can leave it for the sake of having the text "Add review:" there?